### ISSUE TITLE
any plans for Godot 4?

### ISSUE THREAD

**Issue by ghost** [nonameentername/godot-gdnative-csound#1]

**Comment by nonameentername** [comment]
Yes, I'm planning on porting this to work to use GDExtensions instead of GDNative.  I am currently working on getting Fluidsynth working using GDExtensions.  I'll start on csound once I'm done with Fluidsynth.
